.github/workflows/dart.yml

```diff
@@ -0,0 +1,45 @@
+name: Dart
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: windows-latest
+    environment: WindowsEnv
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+
+      - uses: dart-lang/setup-dart@9a04e6d73cca37bd455e0608d7e5092f881fd603
+      - uses: actions/setup-python@v4
+      - uses: microsoft/setup-msstore-cli@v1
+      
+      - name: Configure MS CLI
+        run: msstore reconfigure --tenantId ${{ secrets.AZURE_AD_TENANT_ID }} --clientId ${{ secrets.AZURE_AD_ClIENT_ID }} --clientSecret ${{ secrets.AZURE_AD_CLIENT_SECRET }} --sellerId ${{ secrets.SELLER_ID }}
+
+      - name: Install dependencies
+        run: dart pub get
+
+      # Uncomment this step to verify the use of 'dart format' on each commit.
+      - name: Verify formatting
+        run: dart format --output=none --set-exit-if-changed .
+
+      # Consider passing '--fatal-infos' for slightly stricter analysis.
+      - name: Analyze project source
+        run: dart analyze
+
+      # Your project will need to have tests in test/ and a dependency on
+      # package:test for this step to succeed. Note that Flutter projects will
+      # want to change this to 'flutter test'.
+      - name: Run tests
+        run: dart test
+      
+      - name: Run flutter doctor
+        run: flutter doctor -v

```

Comment:
Is this necessary to publish? Or can we cut it? The Play Store example does not run tests/analysis/doctor

Reply:
It is not necessary. We can remove it.